libtiledbsoma/src/soma/soma_array.h

```diff
@@ -662,14 +662,17 @@ class SOMAArray : public SOMAObject {
      *     same datatype. This argument indicates the number of items in the
      *     value component of the metadata.
      * @param value The metadata value in binary form.
+     * @param force A boolean toggle to suppress internal checks, defaults to
+     *     false.
      *
      * @note The writes will take effect only upon closing the array.
      */
     void set_metadata(
         const std::string& key,
         tiledb_datatype_t value_type,
         uint32_t value_num,
-        const void* value);
+        const void* value,
+        bool force = false);
```

Comment:
Do you still need this `force` toggle now that you are using the SOMA create methods?

Reply:
I do. I checked that this morning.   We have a number of entry points from the R code so this could be a ticket for @mojaveazure and me to resolve "medium-term" but for right now there is sadly no way around this: without it we get massive test failure.